extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -219,473 +219,128 @@ describe('Project: sqlproj content operations', function (): void {
 		should(projFileText.includes(convertSlashesForSqlProj(Uri.file(path.join('$(DacPacRootPath)', 'Extensions', 'Microsoft', 'SQLDB', 'Extensions', 'SqlServer', 'AzureDw', 'SqlSchemas', constants.masterDacpac)).fsPath.substring(1)))).be.true('System db SSDT reference path should be AzureDw');
 	});
 
-	// TODO - make similar test in DacFx
-	// it.skip('Should choose correct msdb dacpac', async function (): Promise<void> {
-	// 	projFilePath = await testUtils.createTestSqlProjFile(baselines.newProjectFileBaseline);
-	// 	const project = await Project.openProject(projFilePath);
 
-	// 	let uri = project.getSystemDacpacUri(constants.msdbDacpac);
-	// 	let ssdtUri = project.getSystemDacpacSsdtUri(constants.msdbDacpac);
-	// 	should.equal(uri.fsPath, Uri.parse(path.join('$(NETCoreTargetsPath)', 'SystemDacpacs', '160', constants.msdbDacpac)).fsPath);
-	// 	should.equal(ssdtUri.fsPath, Uri.parse(path.join('$(DacPacRootPath)', 'Extensions', 'Microsoft', 'SQLDB', 'Extensions', 'SqlServer', '160', 'SqlSchemas', constants.msdbDacpac)).fsPath);
+	it('Should add pre and post deployment scripts as entries to sqlproj', async function (): Promise<void> {
```

Comment:
this diff is making this look like more changes than it really was. The changes in this file were moving the database reference tests down to their own suite and removing some project reloading in the database reference and sqlcmd variable tests